products/appengine/terraform.yaml

```diff
@@ -14,8 +14,8 @@
 --- !ruby/object:Provider::Terraform::Config
 overrides: !ruby/object:Overrides::ResourceOverrides
   FirewallRule: !ruby/object:Overrides::Terraform::ResourceOverride
-    id_format: "{{project}}/{{priority}}"
-    import_format: ["{{project}}/{{priority}}"]
+    id_format: "apps/{{project}}/firewall/ingressRules/{{priority}}"
```

Comment:
Will https://github.com/GoogleCloudPlatform/magic-modules/pull/2461 make this + some other values redundant?